package-lock.json

```diff
@@ -9,9 +9,11 @@
       "version": "1.0.4",
       "license": "Apache-2.0",
       "dependencies": {
-        "@octokit/rest": "^18.5.2",
+        "@0no-co/graphqlsp": "^1.12.11",
+        "@octokit/graphql-schema": "^15.25.0",
         "chalk": "^3.0.0",
         "equatable": "^1.2.0",
+        "gql.tada": "^1.8.2",
```

Comment:
The lockfile seems to be out of sync with package.json (could use another npm install)